cli/client/main.go

```diff
@@ -648,7 +642,7 @@ func mainWithStatusCode() int {
 	if err != nil {
 		log.Fatal().Msgf("%s", err)
 	}
-	req.Proto = "ssh3"
+	req.Proto = "github.com/francoismichel/ssh3"
```

Comment:
Not sure about that one, though, we're setting the `:protocol` HTTP/3 pseudo-header here like it is done fir [webtransport](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3-08#name-creating-a-new-session), or [masque](https://datatracker.ietf.org/doc/html/draft-ietf-masque-connect-udp-15#name-http-2-and-http-3-requests). So I would let that set to `ssh3` for now.